nexus/src/app/update.rs

```diff
@@ -149,4 +157,182 @@ impl super::Nexus {
             .await
             .map_err(HttpError::from)
     }
+
+    /// Get external update status with aggregated component counts
+    pub async fn update_status_external(
+        &self,
+        opctx: &OpContext,
+    ) -> Result<views::UpdateStatus, Error> {
+        let db_target_release =
+            self.datastore().target_release_get_current(opctx).await?;
+
+        let current_tuf_repo = match db_target_release.tuf_repo_id {
+            Some(tuf_repo_id) => Some(
+                self.datastore()
+                    .tuf_repo_get_by_id(opctx, tuf_repo_id.into())
+                    .await?,
+            ),
+            None => None,
+        };
+
+        let target_release =
+            current_tuf_repo.as_ref().map(|repo| views::TargetRelease {
+                time_requested: db_target_release.time_requested,
+                version: repo.repo.system_version.0.clone(),
+            });
+
+        let components_by_release_version = self
+            .component_version_counts(
+                opctx,
+                &db_target_release,
+                current_tuf_repo,
+            )
+            .await?;
+
+        let (blueprint_target, blueprint) = self
+            .quiesce
+            .latest_blueprint()
+            .ok_or_else(|| {
+                Error::internal_error("Tried to get update status before target blueprint is loaded")
+            })?
+            .as_ref()
+            .clone();
+
+        let time_last_blueprint = blueprint_target.time_made_target;
+
+        // Update activity is paused if the current target release generation
+        // is less than or equal to the blueprint's minimum generation
+        let paused = *db_target_release.generation
+            <= blueprint.target_release_minimum_generation;
```

Comment:
Based on the doc comment on `target_release_minimum_generation`, I think this should a `<` and not `<=`, right?